### ISSUE TITLE
ELK stack Open Source (or SSPL): ElasticSearch, Logstash, Kibana

### ISSUE THREAD

**Issue by roberth** [NixOS/nixpkgs#144418]
**Project description**

Upstream has stopped distributing builds of their SSPL-licensed software, which means that we currently only have recent packages for the non-SSPL variants of these packages.

This is a loss, even for non-SSPL users for whom the SSPL-based VM test on ofborg and hydra serves as a canary.

This adds to the other benefits of adding a source-based package.

**Metadata**

* homepage URL: https://www.elastic.co/
* source URL: https://github.com/elastic
* license: sspl
* platforms: unix, linux, darwin, ...

**Comment by ctheune** [comment]
@dpausp here

**Comment by anpin** [comment]
seems like they are opening up again https://www.elastic.co/blog/elasticsearch-is-open-source-again

**Comment by ctheune** [comment]
Yeah. Originally even building the SSPL stuff was troublesome, but having AGPLv3 as an option would make that even more interesting to take a look at again. This requires version 8.16+.
